test/Twig/Tests/Fixtures/filters/replace.test

```diff
@@ -1,8 +1,12 @@
 --TEST--
 "replace" filter
 --TEMPLATE--
-{{ "I like %this% and %that%."|replace({'%this%': "foo", '%that%': "bar"}) }}
+{{ "I liké %this% and %that%."|replace({'%this%': "foo", '%that%': "bar"}) }}
+{{ 'I like single replace operation only %that%'|replace({'%that%' : '%that%1'}) }}
+{{ 'I like %this% and %that%.'|replace(traversable) }}
 --DATA--
-return array()
+return array('traversable' => new ArrayObject(array('%this%' => 'foo', '%that%' => 'bar')))
 --EXPECT--
-I like foo and bar.
+I liké foo and bar.
```

Comment:
So we test multi byte character support as well to prevent accidental regression in the future.